plat/renesas/rcar/rcar_scpi.c

```diff
@@ -83,6 +109,37 @@ static uint32_t scpi_handle_cmd(int cmd, uint8_t *payload_size,
 		mmio_write_32(payload_out + 0x18, 0x0);
 		*payload_size = 0x1c;
 		return SCPI_OK;
+	case SCP_CMD_CLOCKS_CAPS:
+		/* number of implemented clocks */
+		mmio_write_32(payload_out, rcar_clock_nr_clocks());
+		*payload_size = 0x4;
+		return SCPI_OK;
+	case SCP_CMD_CLOCK_GET_INFO:
+		ret = write_clock_info(payload_out, par1 & 0xffff);
```

Comment:
why you mask `par1`?

Reply:
According to '3.2.14 Get Clock Info' from
http://infocenter.arm.com/help/topic/com.arm.doc.dui0922g/scp_message_interface_v1_2_DUI0922G_en.pdf:

[15:0] Clock ID Clock being queried.